content/collections/experiment-apis/en/experiment-management-api-deployments.md

```diff
@@ -27,10 +27,10 @@ Fetch a list of deployments that experiments or flags can be assigned to.
 
 ### Query parameters
 
-|Name|Description|
-|---|----|
-|`limit`| The max number of deployments to be returned. Capped at 1000.|
-|`cursor`| The offset to start the "page" of results from.|
+|Name|Requirement|Type|Description|
+|---|----|----|----|
+|`limit`| Optional | number | The max number of deployments to be returned. Capped at 1000.|
+|`cursor`| Optional | number | The offset to start the "page" of results from.|
```

Comment:
Do these parameters still have types since they are included in the url and not in a request body?
<img width="913" alt="Screenshot 2024-12-17 at 5 33 28 PM" src="https://github.com/user-attachments/assets/c8b686cb-e62a-4a83-a813-ccf25e7643e9" />